apps/yaft/keyboard.cpp

```diff
@@ -343,6 +372,43 @@ getKeyCodeStr(int scancode, bool shift, bool alt, bool ctrl, bool appCursor) {
       case PageDown:
         write_vt_code('6');
         return buf.data();
+
+      case F1:
+        write_f_code('A');
+        return buf.data();
+      case F2:
+        write_f_code('B');
+        return buf.data();
+      case F3:
+        write_f_code('C');
+        return buf.data();
+      case F4:
+        write_f_code('D');
+        return buf.data();
```

Comment:
https://invisible-island.net/xterm/ctlseqs/ctlseqs.html#h3-PC-Style-Function-Keys
```suggestion
      case F1:
        write_ss3('p');
        return buf.data();
      case F2:
        write_ss3('Q');
        return buf.data();
      case F3:
        write_ss3('R');
        return buf.data();
      case F4:
        write_ss3('S');
        return buf.data();
```